deploy/osd-machine-api/management-clusters/011-machine-api.srep-worker-healthcheck.MachineHealthCheck.yaml

```diff
@@ -12,10 +12,12 @@ spec:
       - "infra"
       - "master"
     - key: hypershift.openshift.io/request-serving-component
+      operator: In
       values:
       - "true"
-  # An empty unhealthyConditions causes this MHC to execute only if a machine's underlying node is deleted
-  # https://github.com/openshift/machine-api-operator/blob/c11d6227cb4640ce979edd4e9469342274e88910/pkg/controller/machinehealthcheck/machinehealthcheck_controller.go#L789-L793
-  unhealthyConditions: []
+  unhealthyConditions:
+  - type:    "Ready"
+    timeout: "300s"
```

Comment:
This is when the MHC will consider the node unhealth, so after 5 minutes.  Other non-metal worker MHC is 480s, so this is faster.  What is the rational behind 300s?

Reply:
No rational - I can change this to 480s